### DISCUSSION TITLE
Codec output and "-S"

### DISCUSSION THREAD

**Discussion by waraiauyo** [norkunas/youtube-dl-php#229] [General]
How to choose the output codec ? And what the equivalent of "-S" ( "Sorting" via the yt-dlp documentation ) ?

**Comment by norkunas** [comment]
How to choose.. well it's up to you or yt-dlp, because this library is just a wrapper to allow easily to use it in php :)
Currently sorting option is not implemented here, but contributions are welcome

**Reply by waraiauyo** [reply]
Yeah I just understand recently how the output works x) thanks for ur answer!

**Reply by norkunas** [reply]
I've released new version with that option and many more :)

**Reply by waraiauyo** [reply]
I just saw that thanks! :)
